input/ACTCMB.yml

```diff
@@ -0,0 +1,69 @@
+tracers:
```

Comment:
just a single section with the corresponding mapper

Reply:
Done!